### PR TITLE
acquisition label for mr classifier [GEAR-732]

### PR DESCRIPTION
**Pull request by NPann**
Use acquisition.label for classification. Get precedence over SeriesDescription.

### CODE REVIEW

File: MR_classifier.py [hunk 1]
```diff
@@ -580,14 +580,19 @@ def classify_dicom(dcm, slice_number, unique_iop=''):
...
+        if not classification_dict and series_desc:
+            classification_dict = get_custom_classification(series_desc, '/flywheel/v0/config.json')
         if classification_dict:
             log.info('Custom classification from config: %s', classification_dict)
 
-    # 2. Classification from SeriesDescription
-    if not classification_dict and series_desc:
-        classification_dict = infer_classification(series_desc)
+    # 2. Classification from SeriesDescription or acquisition_label
+    if not classification_dict and (series_desc or acquisition_label):
```
Comment (naterichman) [inline MR_classifier.py:591]: Not clear on why you need the `and` here, you already  have an `if acquisition_label or series_desc` above, right?

File: MR_classifier.py [hunk 2]
```diff
@@ -580,14 +580,19 @@ def classify_dicom(dcm, slice_number, unique_iop=''):
...
+        if acquisition_label:
+            classification_dict = infer_classification(acquisition_label)
+        if not classification_dict and series_desc:
+            classification_dict = infer_classification(series_desc)
```
Comment (naterichman) [inline MR_classifier.py:595]: what the difference between this `if` block and the upper one is?

File: MR_classifier.py (hunk 1 above)
Comment (NPann) [reply MR_classifier.py:593]: #2 is executed if #1 did not populate a `classification_dict` and if `acquisition_label or series_desc` is not None. 
Am I missing something?

File: MR_classifier.py (hunk 2 above)
Comment (NPann) [reply MR_classifier.py:597]: `acquisition_label` has precedence over `series_desc` to define `classification_dcit`. So if `classification_dict` is populated, skip it.

File: MR_classifier.py (hunk 1 above)
Comment (naterichman) [reply MR_classifier.py:593]: It just seems like you're already checking if `acquisition_label or series_desc`in #1.

Comment (NPann) [reply MR_classifier.py]: Ah. I get it know. We could remove that extra condition check. Modified accordingly

**Review by naterichman — APPROVED** [review]

**Pull request merged**